builtin/common/misc_helpers.lua

```diff
@@ -532,7 +532,7 @@ if INIT == "mainmenu" then
 	end
 end
 
-if INIT == "client" or INIT == "mainmenu" then
+if core.gettext then
```

Comment:
```suggestion
if core.gettext then -- for client and mainmenu
```

I think this information should be kept to give a hint that it's not available to mods (which often rely on these common helper functions).